app/src/main/java/com/gsandchase/mileagelogger/MileageCounter.java

```diff
@@ -0,0 +1,70 @@
+package com.gsandchase.mileagelogger;
```

Comment:
This class keeps track of miles for each day. From there, it's easy to total up the miles for a given week/month/year. (I implemented the one for weeks - I'll leave the month/year for you!). You can also add methods to this class to serialize/deserialize to a file.